packages/ipfs-topology/src/ipfs-topology.ts

```diff
@@ -104,24 +105,22 @@ export class IpfsTopology {
     }
     this.logger.imp(`Connecting to peers found in '${url}'`);
     const list = await fetchJson(url);
-    return list || [];
+    return list.map(peer => new Multiaddr(peer)) || [];
```

Comment:
This change fixes the ability to run the daemon from the CLI

Reply:
This will break if `list` is undefined, if it's never undefined then we don't need `|| []`.